OR-197-omit-events-from-multi-select (#1101)
Financial-Times/origami: components/o-multi-select/README.md

oMultiSelect.init(oMultiSelectElement);
```

> `o-multi-select` also dispatches custom event `oMultiSelect.OptionChange` that bubbles and gets triggered when option gets selected or removed. The event data has following interface:

Reviewer: Hey, looks good, just some little suggestions here which might improve readability 😄 Mainly using a heading and short sentences.

```suggestion
### Events 

`o-multi-select` dispatches a custom event `oMultiSelect.OptionChange` that bubbles. It is triggered when an option gets selected or removed. The event data has following interface:
```
